CHANGELOG.md

```diff
@@ -1,12 +1,111 @@
-## v0.5.0 (Unreleased)
+## v0.5.0 (September xx, 2019)
 
  ### Notes
 
+ - This release is not backward compatible with the previous versions of Swarm. Please update your nodes.
+ - Ubuntu PPA support has been removed. We will add support for multiple software distribution platforms at a later stage. In the meanwhile you can still find the Swarm binaries at out [official download page](https://swarm-gateways.net/bzz:/swarm.eth/downloads/).
+
  ### Features
 
+ - **Content Pinning:** You can now pin content to your local swarm node. This way you can guarantee that certain content stays on your node. You can enable pinning by setting the `--enable-pinning` flag. For more information, read the [documentation](https://swarm-guide.readthedocs.io/en/latest/dapp_developer/index.html#pinning-content).
+
+The following features are **experimental** and are **not enabled by default on our public testnet**. If you want to test these features, make sure to use a separate network (e.g. create your own swarm cluster) and set the same feature flags on all nodes.
+
+ - **Push sync:** Additionally to our previous syncing algorithm, pull sync, we now also have "push sync" as an option available to synchronise the content across nodes. If you want to try it out, you have to start you swarm node with `--sync-mode=all`.
+ - **Progress bar via Tags:** [Tags](##put-link-to-tags-docs) allow you to track the progress on content uploads. When using `swarm up`, you can now use the `--track` flag to see the upload progress. This feature requires push sync to be enabled.
+ - **BzzEth:**
+ - **SWAP:**
```

Comment:
@holisticode @vojtechsimetka could you maybe write a sentence or two here, so that we add information about SWAP in the release notes?

Reply:
"A first version of of the SWarm Accounting Protocol (SWAP) is implemented. SWAP is a tit-for-tat incentive system where nodes account how much data they request and serve to their peers. If a node consumes significantly more than it provides, it pays for the consumed data using a L2 payment system: the chequebook. You can try  SWAP on a SWAP enabled network with the flags --swap --swap-backend-url=<evm_endpoint> or learn more in the [SW3 paper](https://www.overleaf.com/read/yszmsdqyqbvc) , [Swap, Swear, and Swindle Games video ](https://www.youtube.com/watch?v=9Cgyhsjsfbg) or [this article](https://www.rifos.org/blog/rif-storage-the-first-chunks/) The economics of this system is still in a highly experimental state. We discourage the usage on mainnet."

Reply:
bzzeth is a new  protocol that enables ethereum and swarm nodes to communicate with each other and help eth nodes distribute blockchain data and state date  over swarm enabling various functionalities like state  syncing via swarm, light client support. This release includes preliminary phases of supporting [block chain on swarm](https://github.com/ethersphere/user-stories/issues/9). This is experimental work that can be trialed with a dedicated version of  the python Trinity client.